config/action.d/pf.conf

```diff
@@ -75,14 +75,13 @@ protocol = tcp
 # Option:  port
 # Notes.:  specifies port to monitor
 # Values:  [ NUM | STRING ]  Default:
-#
-#port = telnet
+port = ssh
```

Comment:
>  > -#port = telnet
>  > +port = ssh
>    But wouldn't this apply then as well for *iptables-common.conf*?
That is exactly what I was saying that many of our actions do carry
defaults for every variable they potentially use, even if we expect them
to be (re)defined from jail action specs.  That is why I thought it
would be nicer to make such (remove default definitions for those which
we expect to be redefined explicitly from the jails) change in 1 step.